### PR TITLE
Remove breadcrumbs

### PR DESCRIPTION
**Pull request by stephendonner**
This needs a good review (as does our continued work to address/redress failures due to changes from the redesign), but I think it's a pretty good start.

Most of the changes are pretty cut-and-dry removals, but I'm not 100% on the following change in particular:

https://github.com/mozilla/Addon-Tests/compare/master...stephendonner:remove-breadcrumbs?expand=1#diff-48b54e45ea0c3687b7105b751b953af9L111

@davehunt @krupa? @mozilla/web-qa-enthusiasts ?

### CODE REVIEW

**Comment by stephendonner** [comment]
I'll also fix the PEP8 issues and squash the merge commits after doing so - sorry 'bout that.

File: pages/desktop/details.py [hunk 1]
```diff
@@ -108,10 +106,7 @@ def _page_title(self):
     def title(self):
         base = self.selenium.find_element(*self._title_locator).text
         '''base = "firebug 1.8.9" we will have to remove version number for it'''
-        if "Themes" in self.selenium.find_element(*self._breadcrumb_locator).text:
```
Comment (stephendonner) [inline pages/desktop/details.py]: Note: not sure the impact (in particular, loss of coverage for Themes) that this removal gives us, but it fixes the test - please review carefully :-)

**Comment by davehunt** [comment]
There's still the following references to breadcrumbs:
* https://github.com/stephendonner/Addon-Tests/blob/remove-breadcrumbs/pages/desktop/base.py#L77-L80
* https://github.com/stephendonner/Addon-Tests/blob/remove-breadcrumbs/pages/desktop/regions/breadcrumbs.py

Does this replace #828? If so, let's close that one.

**Comment by stephendonner** [comment]
I think I've addressed all comments, now - removed the references to breadcrumbs, and the entire breadcrumbs.py file, hopefully.

**Comment by davehunt** [comment]
r+ I'll let @krupa hit the button when she's happy

**Pull request merged**